### DISCUSSION TITLE
Improvement idea: smoothing and brightness settings via json/http

### DISCUSSION THREAD

**Discussion by pimw1** [awawa-dev/HyperHDR#247] [Ideas]
Hi,

I made the transition from Hyperion to HyperHDR and i couldn't be more happier about: it's more stable, better performing, better smoothing controls, more connections options. And the custom component for Home Assistant is better than the Hyperion alternative: Home Assistant need to be restarted first before the Hyperion plugin will work after booting up Hyperion itself. The HyperHDR custom component does not have this issue. To me, that is important, since my Raspberry Pi and ledstrips are completely turned of from main power when i'm not using it.

I've not used the HDR features yet (i am still holding on to my Panasonic Plasma). All in all, HyperHDR is absolutely fabulous!

As always, good implementations lead to more ideas as well. It would be nice if the json/http communication options could be expanded with a smoothing miliseconds setting (in miliseconds), and a brightness setting (0-100).
The reason for this request, is that this will enable to easily set a "viewing profile": laidback viewing (smoothing with more miliseconds and lower brightness) versus turbo viewing (smoothing with only a few miliseconds and higher brightness).

I know that there is an option to disable smoothing altogether, but that is too much: a little bit of smoothing is always prefered imho.

It would be nice if you could consider this.

Regards,

Pim

**Comment by awawa-dev** [comment]
Hi
You can control the brightness already. There is a proper method in the API:
![obraz](https://user-images.githubusercontent.com/69086569/159810782-ea951ff1-0472-497e-a2dc-b2287b89b2d0.png)

When you use the classic calibration it's called the 'Luminance' in the options.  
I will consider implementing new feature to control the smoothing behavior.
Regards
Awawa

**Comment by pimw1** [comment]
Cool! I'm going to use that brightness function for sure. I would be nice if that feature for the smoothing behaviour would be implemented too. Thank you for considering.
